docs/src/main/asciidoc/update-to-quarkus-3.adoc

```diff
@@ -0,0 +1,89 @@
+////
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="update-projects-to-quarkus-3-howto"]
+= Update projects to Quarkus 3 automatically
```

Comment:
I'd like to use the `{quarkus-version}` attribute but am concerned that for now, during this pre-release period, it would show the wrong version number and confuse reviewers. I've created [a follow-up task](https://issues.redhat.com/browse/QDOCS-295) to take care of it post-release.